assets/sass/bulma_styles.scss

```diff
@@ -1,2 +1,26 @@
 @charset "utf-8";
+
+/* GLOBAL VARIABLES */
+$hyper: #3A10E5;
+$white: #FFFFFF;
+$beige: #FFF0E5;
+$primary: $beige;
+
+$body-background-color: $beige;
+$link: black;
```

Comment:
Curious, was the default `$black` not black enough?

Reply:
Oops, I forgot to remove that when I was trying to change the hover state color - I'll remove it now

Reply:
Yeah, I meant why did you use `black` instead of `$black` 😂 
I saw that the default Bulma `$black` is not an absolute black, so I was wondering if it didn't look right or something.

Reply:
If I changed `black` for `$black` I got a variable `$black` not defined error - so I just used `black` as it was easier than trying to debug why that wasn't working:
![image](https://user-images.githubusercontent.com/78568904/140659429-6c8b7a80-c1b5-4f6b-836b-1dca5529e8de.png)